Fails on Windows when Yarn is not installed

Testing on Windows, without Yarn installed,

```
C:\Users\Shinyu>npm create create-app testapp
? description description
? author name MurakamiShinyu
? author email murakami@vivliostyle.org
? template default
? license AGPL-3.0

Creating a new package in C:\Users\Shinyu\create-testapp.

Installing dependencies.
npm ERR! code ENOLOCAL
npm ERR! Could not install from "" as it does not contain a package.json file.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\Shinyu\AppData\Roaming\npm-cache\_logs\2020-08-10T04_11_07_268Z-debug.log
installDeps failed: 1
```

I found that this problem happens when the `npm install` command is called with a `--prefix` option,

https://github.com/uetchy/create-create-app/blob/09dbe7fd17aa4ef8e672aec9a5c4463d429446ea/src/index.ts#L108-L109

and npm has the following bug:

- https://github.com/npm/cli/issues/1290 [BUG] Npm install with --prefix does not work on Windows

Also in Stackoverflow: [NPM prefix flag does not work as expected on windows](https://stackoverflow.com/questions/53146769/npm-prefix-flag-does-not-work-as-expected-on-windows)

--- a/src/index.ts
+++ b/src/index.ts
@@ -106,7 +106,8 @@ async function installDeps(rootDir: string, useYarn: boolean) {
     args = ['install', '--cwd', rootDir];
   } else {
     command = 'npm';
-    args = ['install', '--prefix', rootDir];
+    args = ['install'];
+    process.chdir(rootDir);
   }
   try {
     await spawnPromise(command, args, { stdio: 'inherit' });
@@ -307,7 +308,8 @@ export async function create(appName: string, options: Options) {
         args = ['--cwd', packageDir, 'add', packageName];
       } else {
         command = 'npm';
-        args = ['install', '-D', packageName, '--prefix', packageDir];
+        args = ['install', '-D', packageName];
+        process.chdir(packageDir);
       }
       const child = spawn(command, args, { stdio: 'inherit' });
       child.on('close', (code) => {

